src/python/frigus/solvers/linear.py

```diff
@@ -101,7 +101,8 @@ def solve_equilibrium(m_matrix):
     # ============ done conditioning the linear system ===============
 
     try:
-        x = solve_linear_system_two_step(A, b, n_sub=1)
+        x = solve_lu_mp(A, b)
```

Comment:
why did you use the ``mp`` solver?

Reply:
because with the data by lique we don't need to solve with the method by francesca, in order to avoid to have 0 population for j=1 and all the odd j levels